packages/macro/index.d.ts

```diff
@@ -163,46 +188,108 @@ export function select(value: string, choices: ChoiceOptions): string
  *
  * @param descriptor The message descriptor
  */
-export function defineMessage(descriptor: MessageDescriptor): MessageDescriptor
+export function defineMessage(
+  descriptor: MacroMessageDescriptor
+): MessageDescriptor
 
-export type TransProps = {
+type CommonProps = {
   id?: string
   comment?: string
-  values?: Record<string, unknown>
   context?: string
-  children?: React.ReactNode
-  component?: React.ComponentType<TransRenderProps>
   render?: (props: TransRenderProps) => ReactElement<any, any> | null
   i18n?: I18n
 }
 
-export type ChoiceProps = {
-  value?: string | number
-} & TransProps &
-  ChoiceOptions<ReactNode>
+type TransProps = {
+  children: ReactNode
+} & CommonProps
+
+type PluralChoiceProps = {
+  value: string | number
+  /** Offset of value when calculating plural forms */
+  offset?: number
+  zero?: ReactNode
+  one?: ReactNode
+  few?: ReactNode
```

Comment:
Why is there the case `two` missing here?

Reply:
Good point, it wasn't there in the first place i discovered it while comparing to the docs.